src/features/session_replay/aggregate/index.component-test.js

```diff
@@ -289,24 +263,10 @@ describe('Session Replay', () => {
       const [harvestContents] = sr.prepareHarvest()
       expect(harvestContents.qs).toMatchObject({
         protocol_version: '0',
-        // content_encoding is omitted when the payload is not compressed
         browser_monitoring_key: info.licenseKey
       })
-      expect(harvestContents.qs.content_encoding).toBeUndefined()
-      expect(harvestContents.body).toMatchObject({
-        type: 'SessionReplay',
-        appId: info.applicationID,
-        timestamp: expect.any(Number),
-        blob: expect.any(String),
-        attributes: {
-          session: session.state.value,
-          hasSnapshot: expect.any(Boolean),
-          hasError: expect.any(Boolean),
-          agentVersion: expect.any(String),
-          isFirstChunk: expect.any(Boolean),
-          'nr.rrweb.version': expect.any(String)
-        }
-      })
+      expect(harvestContents.qs.attributes.includes('content_encoding')).toEqual(false)
```

Comment:
There's no check in this file for cases where `content_coding` is included. That should probably existed at least somewhere.

Reply:
All the tests that check `anyQuery` are checking for content_encoding already, for example: https://github.com/newrelic/newrelic-browser-agent/pull/588/files#diff-9eed0f14ee8a3eecadf0caadd2f3ca623ab18fbf8acbe6c23f8439b6bfb9df47R232

Reply:
added `expect(harvestContents.qs.attributes.includes('content_encoding=gzip')).toEqual(true)` to the compressed test case